openmdao/core/system.py

```diff
@@ -17,7 +17,20 @@
 import numpy as np
 import networkx as nx
 
+try:
+    from IPython.display import HTML, display
```

Comment:
Seems like we don't need both the `ipython` flag and the `tab_pkg` flag. They are always tested in conditional together (i.e., there is no special action when one it True and the other False). I think you can squash these down to one flag that is basically the same as `ipython and tab_pkg`

I think you could even get fancier and have no flags. Just set `tabulate` to None if both ipython and tabulate fail to import, and then use `if tabulate` for the conditionals.

Reply:
Fixed